packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -64,7 +64,11 @@ techdocs:
 
 auth:
   # see https://backstage.io/docs/auth/ to learn about auth providers
-  providers: {}
+  providers:
+    github:
+      development:
+        clientId: ${AUTH_GITHUB_CLIENT_ID}
+        clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
```

Comment:
Not sure we necessarily want to add this in the end, was it needed for the GitHub module? In practice this is something to add to `app-config.production.yaml` for both `development` and `production` envs and then potentially have another setup for in `app-config.local.yaml`